src/test_utils/kbn_server.ts

```diff
@@ -60,7 +60,10 @@ const DEFAULT_SETTINGS_WITH_CORE_PLUGINS = {
   },
 };
 
-export function createRootWithSettings(...settings: Array<Record<string, any>>) {
+export function createRootWithSettings(
+  settings: Record<string, any>,
+  cliArgs: Partial<CliArgs> = {}
```

Comment:
I had to hack `createRoot` to be able to specify env vars because additional plugins can be used in `dev` mode only.
even more interesting that we run `integration_tests` in `production` mode, which is ok, but functional tests in `development`. what the difference?

Reply:
Yeah, agree, we should unify environment these tests run in at some point.